src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/SearchServiceIntTest.kt

```diff
@@ -466,7 +466,7 @@ class SearchServiceIntTest : IntegrationTestBase() {
     val searchingPerson = Person(
       firstName = firstName,
       lastName = "Smith",
-      addresses = listOf(Address(postcode = "LS2 1AB"), Address(postcode = "LD2 3BC")),
+      addresses = listOf(Address(postcode = "LS1 2AB"), Address(postcode = "LD2 3BC")),
```

Comment:
The name of this test still refers to Levenshtein

Reply:
I'll update in subsequent PR